test/integration/manifests/cns/ciliumconfigmap.yaml

```diff
@@ -24,5 +24,8 @@ data:
       "ChannelMode": "CRD",
       "InitializeFromCNI": false,
       "ManageEndpointState": true,
-      "ProgramSNATIPTables" : true
+      "ProgramSNATIPTables" : true,
+      "EnableCNIConflistGeneration": true,
+      "CNIConflistFilepath": "/etc/cni/net.d/cilium.conflist",
```

Comment:
this should usually be fine, but why did we drop the priority prefix?
`05-cilium.conflist`

Reply:
ah just copied over the path I wrote when I was testing locally. I can add the prefix